lib/libzfs/libzfs_pool.c

```diff
@@ -3161,6 +3163,8 @@ zpool_vdev_online(zpool_handle_t *zhp, const char *path, int flags,
 	if (avail_spare)
 		return (zfs_error(hdl, EZFS_ISSPARE, msg));
 
+#ifndef __FreeBSD__
+	char *pathname;
 	if ((flags & ZFS_ONLINE_EXPAND ||
 	    zpool_get_prop_int(zhp, ZPOOL_PROP_AUTOEXPAND, NULL)) &&
 	    nvlist_lookup_string(tgt, ZPOOL_CONFIG_PATH, &pathname) == 0) {
```

Comment:
Unrelated to this commit, but pathname seems no longer used here, while in FreeBSD 12 I see it was passed to zpool_relabel_disk().

Reply:
Indeed. My original commit here also tried to use pathname and pass it to zpool_relabel_disk(), but that caused some test failures on Linux, so I've left it alone for now.